ext_mongodb.php

```diff
@@ -210,7 +210,7 @@ function($value) {
 <<__NativeData("MongoDBDriverManager")>>
 class Manager {
 	<<__Native>>
-	public function __construct(string $dsn = "mongodb://127.0.0.1/", array $options = array(), array $driverOptions = array());
+	public function __construct(string $dsn = "", array $options = array(), array $driverOptions = array());
```

Comment:
Up to you I suppose. If I end up adding a test to PHPC for empty strings, feel free to assign me the task of fixing this in the HHVM driver.